triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/TriggerRunnerConfig.scala

```diff
@@ -48,6 +49,7 @@ object TriggerRunnerConfig {
       // 256 here comes from the default ExecutionContext.
       submissionFailureQueueSize = 256 + parallelism,
       maximumBatchSize = 1000,
+      batchingDuration = 250.milliseconds,
```

Comment:
We batch messages into groups of 1000 every 250ms. Whichever limit is hit first determines when trigger evaluation occurs for the batch.